### PR TITLE
NXP-32252: admin-console-nuxeo-server-information

### PR DESCRIPTION
**Pull request by Nishant0928**
https://jira.nuxeo.com/browse/NXP-32252

### CODE REVIEW

File: src/app/features/admin-home/store/effects.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const loadVersionInfoEffect = createEffect(
+  (actions$ = inject(Actions), adminHomeService = inject(AdminHomeService)) => {
+    return actions$.pipe(
+      ofType(AdminHomeActions.fetchProbesInfo),
+      switchMap(() => {
+        return adminHomeService.getProbesInfo().pipe(
+          map((data) => {
+            return AdminHomeActions.fetchProbesInfoSuccess({
+              probesInfo: {
+                version: data.version,
```
Comment (swarnadipa-dev) [inline src/app/features/admin-home/store/effects.ts:17]: use ?. to check for null/undefined property value

File: src/app/features/admin-home/store/reducers.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export const adminHomeReducer = createReducer(
+  initialState,
+  on(AdminHomeActions.fetchProbesInfo, (state) => ({
+    ...state,
+    error: null,
+  })),
+  on(AdminHomeActions.fetchProbesInfoSuccess, (state, { probesInfo }) => ({
+    ...state,
+    probesInfo: {
+      version: probesInfo.version,
```
Comment (swarnadipa-dev) [inline src/app/features/admin-home/store/reducers.ts:29]: use ?. to check for null/undefined property value

File: src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
+export class AdminRegistrationVersionComponent implements OnInit {
+  probesInfo$: Observable<probesInfo>;
+  error$: Observable<any>;
+
+  constructor(
+    private store: Store<{ admin: AdminState }>,
+    private adminHomeService: AdminHomeService
+  ) {
+    this.probesInfo$ = this.store.pipe(
+      select((state) => state.admin.probesInfo)
```
Comment (swarnadipa-dev) [inline src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts:23]: use ?. to check for null/undefined property value

**Review by swarnadipa-dev — COMMENTED** [review]
change interface file name to probes-info.interface.ts

File: src/app/features/admin-home/store/effects.ts (hunk 1 above)
Comment (Nishant0928) [reply src/app/features/admin-home/store/effects.ts]: done

File: src/app/features/admin-home/store/reducers.ts (hunk 2 above)
Comment (Nishant0928) [reply src/app/features/admin-home/store/reducers.ts]: done

File: src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts (hunk 3 above)
Comment (Nishant0928) [reply src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts]: done

File: src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.scss [hunk 4]
```diff
@@ -14,6 +14,23 @@ mat-card.mat-mdc-card:not([class*="mat-elevation-z"]) {
...
 }
+.mat-mdc-card-content {
+  display: block;
+  padding: 0 0 !important;
+}
 .mat-mdc-card-content:first-child {
   padding-top: 0;
 }
+
+.cluster {
```
[the remark this answers is not in the document]
Comment (Nishant0928) [reply src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.scss]: done

**Review by swarnadipa-dev — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

File: src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts [hunk 5]
```diff
@@ -1,8 +1,31 @@
-import { Component } from "@angular/core";
```
Comment (madhurkulshrestha-hyland) [inline src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts:1]: Unit tests for the changes ?

**Review by madhurkulshrestha-hyland — CHANGES_REQUESTED** [review]

File: src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts (hunk 5 above)
Comment (Nishant0928) [reply src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts:1]: Th unit test would be covered in the https://github.com/nuxeo/nuxeo-admin-console-ui/pull/20/files in a separate Pr

File: src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.html [hunk 6]
```diff
@@ -1,4 +1,15 @@
-<mat-card role="region" aria-label="registration-version">
-    <mat-card-content>Registration and Version Info</mat-card-content>
-    <button mat-flat-button color="primary">Details</button>
-</mat-card>
\ No newline at end of file
+<mat-card role="region" aria-label="registration-version" class="registration-version registration-version--mdc">
```
[the remark this answers is not in the document]
Comment (Nishant0928) [reply src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.html]: changes done

**Pull request merged**